docs/src/main/sphinx/develop/event-listener.rst

```diff
@@ -9,9 +9,12 @@ events:
 * Query completion (success or failure)
 * Split completion (success or failure)
 
+Event details include session, query execution, resource utilization, timeline,
+and more. Use the :doc:`/admin/web-interface` to inspect the query details interactively. 
+
 This functionality enables development of custom logging, debugging and
 performance analysis plugins. In a Trino cluster, only a single event
-listener plugin can be active at a time .
+listener plugin can be active at a time.
```

Comment:
Unrelated to this PR but @hashhar and @electrum  .. isnt that wrong?

Reply:
Good catch. Yes, we support multiple listeners now.